terra/assetlist.json

```diff
@@ -6509,7 +6509,55 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.png",
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.svg"
         }
+      ],
+      "base": "cw20:terra1m58hc296srr6xygrrfyu6u0e32pl8d459nfs55pmegke8zlv94rqsjz882",
```

Comment:
You need separate asset objects (currently all the properties are mixed into a single object